src/main/test/com/social/VkPostTest.java

```diff
@@ -0,0 +1,33 @@
+package com.social;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+import com.social.interfaces.ISocialNetworkService;
+import com.social.models.Post;
+import com.social.models.SocialNetwork;
+import com.social.models.SocialNetworkType;
+import com.social.services.SocialNetworkFactory;
+import com.social.utils.PropertyService;
+
+public class VkPostTest {
+	private String vkLogin = PropertyService.getValueProperties("vkLogin");
+	private String vkPass = PropertyService.getValueProperties("vkPass");
+	
+	@Test
+	public void test(){
```

Comment:
Name of the test should show method or situation that you are testing. Modify it